Microsoft Edge 148 web platform release notes (May 2026) (#3772)
MicrosoftDocs/edge-developer: microsoft-edge/web-platform/release-notes/148.md

  <div><input></div>
</div>
```

Previously, `@container` required some query in addition to the name.

See also:
* [@container](https://developer.mozilla.org/docs/Web/CSS/Reference/At-rules/@container) at MDN.
* [container-name](https://developer.mozilla.org/docs/Web/CSS/Reference/Properties/container-name) at MDN.

<!-- ------------------------------ -->
#### The `revert-rule` keyword

The `revert-rule` keyword rolls back the cascade to the previous rule, similar to how `revert-layer` rolls back the cascade to the previous layer.  For example:

```css
div { color: green; }
div { color: revert-rule; /* Effectively green */ }
```

This is especially useful in combination with conditionals, as it allows eliminating the current rule if some condition is not met:

```css
div {
  display: if(style(--layout: fancy): grid; else: revert-rule);
}
```

See also:
* [The revert-rule keyword](https://drafts.csswg.org/css-cascade-6/#revert-rule) in _CSS Cascading and Inheritance Level 6_.

<!-- ------------------------------ -->
#### `at-rule()`: CSS feature detection

The `at-rule()` function can be used in CSS `@supports` to feature-detect support for CSS at-rules.

See also:

> * [The at-rule() function](https://drafts.csswg.org/css-conditional-5/#at-rule-function) in _CSS Conditional Rules Module Level 5_.

Reviewer: The current link doesn't appear to link to a sub portion of the document. What about:
```suggestion
* [The at-rule() function](https://drafts.csswg.org/css-conditional-5/#funcdef-supports-at-rule) in _CSS Conditional Rules Module Level 5_.
```

Reviewer 2: What's the best link text, url, & link annotation? 

Suggested above: 
https://drafts.csswg.org/css-conditional-5/#funcdef-supports-at-rule
same page, a different anchor, that exists

https://review.learn.microsoft.com/en-us/microsoft-edge/web-platform/release-notes/148?branch=pr-en-us-3772#at-rule-css-feature-detection

@captainbrosset

Author: ```suggestion
* [Extensions to the @supports rule](https://drafts.csswg.org/css-conditional-5/#at-supports-ext) in _CSS Conditional Rules Module Level 5_.
```

By defining the relationships between axes mathematically within the avar version 2 table, fonts can achieve complex designs with fewer masters, leading to smaller file sizes.

See also:
* [Variable fonts](https://developer.mozilla.org/docs/Web/CSS/Guides/Fonts/Variable_fonts) at MDN.

<!-- ------------------------------ -->
#### Pointer event suppression on drag start

When a drag operation starts, the browser now sends `pointercancel`, `pointerout`, and `pointerleave` events to the drag source, to indicate that the pointer event stream has ended.

This behavior ensures that the drag source doesn't expect any more events from this pointer during the drag operation.

This behavior is now fully implemented across all platforms.

See also:
* [HTML Drag and Drop API](https://developer.mozilla.org/docs/Web/API/HTML_Drag_and_Drop_API) at MDN.
* [Pointer events](https://developer.mozilla.org/docs/Web/API/Pointer_events) at MDN.

<!-- ------------------------------

> todo: verify if this is also shipping in Edge.

Reviewer 2: Is the Prompt API also shipping in Edge?

@captainbrosset
